src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs

```diff
@@ -108,7 +108,7 @@ public async Task ExecuteCommandAsync(ListPackageArgs listPackageArgs)
                             {
                                 var printPackages = true;
 
-                                // Handle outdated
+                                // Handle --outdated
                                 if (listPackageArgs.IncludeOutdated)
```

Comment:
switch?

Reply:
Not sure what you mean with this comment...